src/model_bias_analysis.py

```diff
@@ -255,6 +256,43 @@ def per_term_negative_rates(df, terms, model_families, threshold, text_col,
         records.append(term_record)
     return pd.DataFrame(records)
 
+### Summary metrics
+def diff_from_base(base_metrics, per_term_results, model_families, metric_column):
+    diffs = {}
+    for fams in model_families:
+        family_name = model_family_name(fams)
+        family_base_metrics = base_metrics[family_name]
+        metric_diff_sum = 0.0
+        for metric_list in per_term_results[family_name + metric_column]:
+            for i in range(10):
+                metric_diff_sum += abs(family_base_metrics[i] - metric_list[i])
```

Comment:
Let's not hardcode 10. Something like this instead?

```
per_model_metric_diffs = [abs(overall_score - per_term_score)
                                          for overall_score, per_term_score
                                          in zip(family_base_metrics, metric_list)]
diffs[family_name] = sum(per_model_metric_diffs)

```

Reply:
Good point, this is much nicer.